src/SFA.DAS.Notifications.Application/Commands/SendSms/SendSmsCommandHandler.cs

```diff
@@ -16,23 +18,33 @@ namespace SFA.DAS.Notifications.Application.Commands.SendSms
     public class SendSmsCommandHandler : AsyncRequestHandler<SendSmsCommand>
     {
         [QueueName]
+#pragma warning disable IDE1006 // Naming Styles
         public string send_notifications { get; set; }
+#pragma warning restore IDE1006 // Naming Styles
 
         private readonly ILog _logger;
         private readonly INotificationsRepository _notificationsRepository;
         private readonly IMessagePublisher _messagePublisher;
+        private readonly ITemplateConfigurationService _templateConfigurationService;
 
-        public SendSmsCommandHandler(INotificationsRepository notificationsRepository, IMessagePublisher messagePublisher, ILog logger)
+        public SendSmsCommandHandler(
+            INotificationsRepository notificationsRepository,
+            IMessagePublisher messagePublisher,
+            ITemplateConfigurationService templateConfigurationService,
+            ILog logger)
         {
             if (notificationsRepository == null)
                 throw new ArgumentNullException(nameof(notificationsRepository));
             if (messagePublisher == null)
                 throw new ArgumentNullException(nameof(messagePublisher));
+            if (templateConfigurationService == null)
```

Comment:
Really?

Reply:
Personally I wouldn't have any of the null checks, so I had to choose between justifying removing them, justifying including a check for my own, or justifying breaking the pattern.